tests/app/main/views/test_services.py

```diff
@@ -239,6 +221,29 @@ def test_service_view_shows_info_banner_for_removed_and_private_services(self, d
             )
         ]
 
+    @pytest.mark.parametrize('service_status', ['disabled', 'enabled'])
+    def test_info_banner_contains_publish_link_for_ccs_category(self, data_api_client, service_status):
```

Comment:
Nice work on the test coverage, but I think we're missing a test that there is NO publish link for regular admins.

Reply:
Hmmm Let me look into this.